dev_tools/docs/build_api_docs.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 The Cirq Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tool to generate external api_docs for Cirq (Shameless copy from TFQ)."""
+
+from __future__ import absolute_import
```

Comment:
are these future imports necessay since we don't support python < 3.6?

Reply:
+1